_data/icons/alyx.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://bafkreifd43fcvh77mdcwjrpzpnlhthounc6b4u645kukqpqhduaveatf6i",
+    "width": 300,
+    "height": 300,
+    "format": "png"
```

Comment:
identify bafkreifd43fcvh77mdcwjrpzpnlhthounc6b4u645kukqpqhduaveatf6i
bafkreifd43fcvh77mdcwjrpzpnlhthounc6b4u645kukqpqhduaveatf6i PNG 2481x2481 2481x2481+0+0 8-bit sRGB 79740B 0.000u 0:00.000

Reply:
change width to 2481?